python/doc/examples/meta_modeling/overfitting_model_selection.ipynb

```diff
@@ -0,0 +1,860 @@
WlNDWrQghaJ2yk8HB48gIwZTY/HqneJAcA2rXUxYJvteVzhQB+tdQmgItBVKVUyxvpJWusytsdG+4csRMooFL3oRU//s3DsGGzebHVIwgx37hCxfSsz3vmTlrS0OhohhB0VLAhHKE0lfqIOG/mSdqThIQULWh2ZeJxxjOMylxOu5x2fqF5vZf4sxVY1vusDC2zfLwAaxN1Aa/2X1voX2/e3gWPA0w6LUAg7aEQjjqc7yx8Tu0KvXvDwodUhiZRav55j/8tBixzdE5dDKIRwWaMDFRkyaq7gjS/BeHOZDR71GDPkrtWhiQRsZSuTmcwKVuCFV+J3tFPKCVjX+PbWWv8FRiMbHl8QVylVGHgF+DnG4m5KqUNKqbnxpa0I4YzSkpbudGdY3X3wzDPw+edWhyRS6P6S+Xze/F+60MXqUIQQdubnB7NnKfIXCiNUZaBTwRUUrexNswW1pZKVEwohhJa05Cu+ogAFkraznQZbAqSxy1EBpdRWIF88q5I0LEEplRlYBfTUWkf9z/4CGImRbDUSmAC0TWD/DkAHAG9vb4KTMcvgnTt3krWfK5Brc7ySniUJrBjIxjZDqd75Y/YVKsTDHEn/+9FZr88MrnJtaW7f5tXvt3NjTHWOBB9J1D6ucm1CiPj5+YGfnxfzmc8nfEKeyJ+h2yCoUQM2bYJk3M9TrYgI+PtvuHoVSpc26jea5D73eY/36EMfqlI16QewY8+33RrfWuvqCa1TSl1WSuXXWv+llMoPXElgu7QYDe8grfXqGMe+HGOb2cD6x8QxC5gF4OPjo319fZN6KQQHB5Oc/VyBXJs1OtCBLc0vUmd/Oypv2ABz5iT5GM58fSnlKtf2YN4MtlRXjCgxmWIliiVqH1e5NiHE47WhDXvYQxuPtqz6fCWqT1+oVg22bZMG+JMcOQKdOsG+fZAzJ6RPb/zMRo+Gt94yJc/aH38KUYg+JLPAgR17vq1KO1kLtLZ93xr4Ju4GSikFfAkc01pPjLMuf4ynDYHEdTkJ4ST88WcBC7g+tAds3GjcgITLubJkMseavUwxEtfwFkK4l8lM5iIX+VSNhwkTwNcX6tQxek3Fox4+NOa6qFoV2rQxGrh//QVnzhj1Gnv2hNq14f79FJ1mLnP5gR+YxzwUyWzIu2HO9xighlLqFFDD9hyl1FNKqajKJZWBlsCb8ZQUHKeUOqyUOgRUBXo5OH4hUqQABahHPb7IFsRP74zml9d64KkiZbY0FxJx5S+y7T3Ja3U/sToUIYRF0pGOVaxiEpPYqrYZDfCXXoJ69eDePavDcy4PH0L9+rBrF/zyC7RvD+nSGeuUgvfeM3rEs2aFzp2NWYySYT/76U9/VrOaLKSg8exuPd9a62ta62pa62K2r//all/SWtexfb9La6201i/FLSmotW6ptS5tW1cvavCmEK6kL30ZG3SRGotaEhamaclCmS3NhRxeMZSf3s5BxYxvWh2KEMJCz/AMS1iCH36cUqfhiy+gQAFo1MioFS0gMtLo6fb0hA0bjIID8UmTBubOhQMHYPr0JJ/mEpdoSENmMYsSlEhZzG7Y8y1EqvciLxIWMIy79zzpxjQ+YSDZuCGzpbmKpcvI3PzD5H+k6SYSM2mabbtztk8sDyql9js6TiHsyRdfRjCCetTjhsctowEZGWnkNSezB9dtaG2U1g0JgeXLjQb242TODGvWwIgRsHNnok9zj3s0oAGd6ERDGqYwaIzGtzv1fAshDA9CjCqbB/BhLfUYzjBAZktzdgfOraLgsbtUrDnU6lCcQWImTYtS1fYppo9jQhPCcTrSkepUpznNiUjrAStWwMGDRiMyNZsxA7Zvh3XrIEMi50IoWtQoRNC+vVER5Qk0mva0pyhFGcSgFAZsc+uW9HwL4Y6eiTEr2iBG05wllOaQzJbm5E4tHMLFZq/h6SWT6pCISdOESC0mMYmHPKQf/Ywe3A0bYMECoyc8Nfr9dxg6FFatguzZk7Zv3bqQJw8sXfrETccwhpOcZC5zzfs00o4933YrNSiEeLLRgYq2HcIJC03Dv+RiKCP4wqMr50b9AKk8ncFZndInqbDwBPmWzLA6FGcRa9I0pVRCk6Zp4DullAZm2srAPkLmZng8d742cI/r65GmB11e7YJniCd1/q5Dxo8/poy/P179+xNsdXB2Et/rpsLCKNulC3+2acNfly7BpUtJPm6OBg0oNnAge/PlM/LF47Er1y4mPz+Z6Qem83PYz/FukxwlTp3i35w5uRwcbPr/S2l8C2EhPz+IxIO2AX8SEfIU3z3zIYGeX1I5chHQyurwRDzW/NiXNl45yeDzutWhOIxJk6ZV1lpfsjXOtyiljmutf4i7kczN8HjufG3gPtf3PM9T5YUq1HihBuVpBfny4dOsGV4HDsCzz1odnunifd369oWXXqL4uHEUT27d7ipV+GfWalY3u8K0a80pWBACA433ToC97GUyk9nIRsr9r1yKruEREyfiXb48JXx9Tf9/KWknQlispZ8HU86t5Z3IBpw570mu5V9A//5w/brVoYk4LnGJ/PM3k6l1F1MmgXAVWuvqWusX43l8A1yOmnvhcZOmaa0v2b5eAdYA5R0VvxCO9gIvsJKV+OHHIQ5BzZqcb9EC3nknddQA37ULliyB2bNTdK8M+krRPmQYna6OROmIWBXBznCGBjRgLnMph8kNb3C/UoNCiNiiZko7xjHw8YGGDWHwYKvDEnFMDR3Le6sUmVp0sDoUZ5KYSdMyKaWyRH0P1EQmRxNu7nVeZypTqUtdLnKRPxs2hP/9zyi5584VUMLCoGNHmDwZcuVK0aECAmDtg5rcJBvvsQqA0FAYEBBBbWozmMG8wztmRP0oKTUohHvLQAa60pXxjDcWBAbC6tWwXyqyOYt/+ZerX3+JqlARnn7a6nCcSWImTfMGdimlfgP2Ahu01pssiVYIB2pKU3rQgzrU4U6auzB1Kvz5J4wZY3Vo9jN+vJFa8957KT6UUflL8Rk9ac+c6OUXQ
```

Comment:
Suggestion rejected by Github
    *We see that the RMSE on the train set continuously decreases, reaching zero when the polynomial degree is such that the number of coefficients is equal to the train dataset sample size. In this extreme situation, the least squares solution is equivalent to solving a linear system of equations: this leads to null residuals.*